### PR TITLE
ci(travis): use travis to release

### PR DESCRIPTION
**Pull request by tglatt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,52 +6,53 @@ git:
 
 #
 
+.env:
+  github_keys: &github_keys
+    - secure: "Zp76yFJIGFBRC4qFAGWNIyOHBSj8S/sKhAehLsYCTRPKWW4o30hg1jMhwp6XaWeyug9jRx+5CNevytr79p7XxRcKKWr1VQ4VWhg9mSSDeg0tNVZGNcu4z6kaDjfsEgHcYcNmgnJ4epo4Z8dZfJ4uCcD9mBrNcYGybiCAVFEok1LXXrUvgdUa/FgbPZGbMkG3EhR+8X7Z/8lLO/vkMpuYvpeTkrItck3PZxqjGIecTTvV/jfOB60SWRrlntTG0nDrtPCrHDHUy3rM43IFXMONzRlX9sqY8jv7yC4aImgLtnwg/Ii9hf+xoeL7+LpwIfG/+0GSBWnWR8pWSl7qgmRbsGZe32S/f0kIiAUaGaNpTzXyN07BP7mEIygLyCZETbBucafOlymb05ljsitXsYalogOuAHvkmkIeqTowTtqaVqnDUlSM9cRth31y0UIBeT3IIQr8kQhzFLVAKucWp4B1cP8gaC8jKsaI9dyPwN9ZvIoZsQCm7aWb7AGnVgg148byIXIK6QS9qen51YVTnRoPuJylABl6O6L6JV62o5aav8QUsV+UYKu/48wDkEI+y8oWfvEC+mdEZaOzUGm4lp4ucRCM/s+J3wtaPcyNG/SvOq4bX1+cgbP49UjxQkzvk2G9uzXbgwENEA9v4xCGewOt/PDyd/FTBTw/Qsalatpl8/o="
```
Comment (douglasduteil) [inline .travis.yml:11]: Is it a new generated key ?

File: README.md [hunk 2]
```diff
@@ -93,3 +93,39 @@ Liste des synchronisations:
 - Règle 5 - `without_continuation` (sans suite)
     - Messages de l'usager envoyés après la date `processed_at` et dossier archivé
 
+
+## Release policy
```
Comment (douglasduteil) [inline README.md]: strange indentation

**Review by douglasduteil — APPROVED** [review]
not bad ;)

File: .travis.yml [hunk 3]
```diff
@@ -6,52 +6,53 @@ git:
...
   language: node_js
   node_js: "10"
+  cache: yarn
   before_install:
     - curl -o- -L https://yarnpkg.com/install.sh | bash
     - export PATH="$HOME/.yarn/bin:$PATH"
   install:
     - yarn --frozen-lockfile
 
-.environments:
```
Comment (tglatt) [inline .travis.yml]: @douglasduteil , It is this key!!

File: .travis.yml [hunk 4]
```diff
@@ -6,52 +6,53 @@ git:
...
-  - &github_keys
-    - secure: "Zp76yFJIGFBRC4qFAGWNIyOHBSj8S/sKhAehLsYCTRPKWW4o30hg1jMhwp6XaWeyug9jRx+5CNevytr79p7XxRcKKWr1VQ4VWhg9mSSDeg0tNVZGNcu4z6kaDjfsEgHcYcNmgnJ4epo4Z8dZfJ4uCcD9mBrNcYGybiCAVFEok1LXXrUvgdUa/FgbPZGbMkG3EhR+8X7Z/8lLO/vkMpuYvpeTkrItck3PZxqjGIecTTvV/jfOB60SWRrlntTG0nDrtPCrHDHUy3rM43IFXMONzRlX9sqY8jv7yC4aImgLtnwg/Ii9hf+xoeL7+LpwIfG/+0GSBWnWR8pWSl7qgmRbsGZe32S/f0kIiAUaGaNpTzXyN07BP7mEIygLyCZETbBucafOlymb05ljsitXsYalogOuAHvkmkIeqTowTtqaVqnDUlSM9cRth31y0UIBeT3IIQr8kQhzFLVAKucWp4B1cP8gaC8jKsaI9dyPwN9ZvIoZsQCm7aWb7AGnVgg148byIXIK6QS9qen51YVTnRoPuJylABl6O6L6JV62o5aav8QUsV+UYKu/48wDkEI+y8oWfvEC+mdEZaOzUGm4lp4ucRCM/s+J3wtaPcyNG/SvOq4bX1+cgbP49UjxQkzvk2G9uzXbgwENEA9v4xCGewOt/PDyd/FTBTw/Qsalatpl8/o="
```
Comment (tglatt) [inline .travis.yml:20]: @douglasduteil, it is this key!

Comment (douglasduteil) [reply .travis.yml:20]: :thinking:

Comment (douglasduteil) [reply .travis.yml:20]: Let merge this and release :)
I will look at the Travis build info to ensure that it's the right one :eye:

**Pull request merged**